ietf/doc/views_status_change.py

```diff
@@ -465,7 +465,7 @@ class StartStatusChangeForm(forms.Form):
     document_name = forms.CharField(max_length=255, label="Document name", help_text="A descriptive name such as status-change-md2-to-historic is better than status-change-rfc1319.", required=True)
     title = forms.CharField(max_length=255, label="Title", required=True)
     ad = forms.ModelChoiceField(Person.objects.filter(role__name="ad", role__group__state="active",role__group__type='area').order_by('name'), 
-                                label="Shepherding AD", empty_label="(None)", required=True)
+                                label="Shepherding AD", empty_label="(None)", required=False)
```

Comment:
This has to come with a migration. (The tests should really complain if migrations are missing).

Reply:
Nevermind - reading too fast and was thinking models rather than forms.